global/scripts/semgrep/run.sh

```diff
@@ -11,23 +11,46 @@ fileName="$CONTAINER_PATH/$REPORT_PATH/semgrep.json"
 
 defaultFile="$SCRIPTS_DIR/global/scripts/semgrep/.semgrepignore"
 cp "$defaultFile" .
-
 if [ -f ".semgrep.yaml" ]; then
```

Comment:
Could you please invert this IF putting the .semgrepcustom first? And please, make 1 line of comment for each IF you add explaining why you added.

Reply:
I reversed the order and separated them